app/views/index/reader.phtml

```diff
@@ -7,8 +7,7 @@ call_user_func($this->callbackBeforeEntries, $this);
 $lazyload = FreshRSS_Context::$user_conf->lazyload;
 $content_width = FreshRSS_Context::$user_conf->content_width;
 ?>
-
```

Comment:
Please keep it.

Reply:
fixed,  but I do not understand why it is necessary